dali/operators/imgcodec/peek_image_shape.cc

```diff
@@ -115,7 +115,7 @@ void ImgcodecPeekImageShape::RunImpl(Workspace &ws) {
 }
 
 
-DALI_REGISTER_OPERATOR(experimental__PeekImageShape, ImgcodecPeekImageShape, CPU);
```

Comment:
We probably want to keep `experimental.peek_image_shape` for backwards compatibility as well, right?